src/Microsoft.Health.Fhir.Shared.Api/Features/Filters/WeakETagBinder.cs

```diff
@@ -22,7 +22,15 @@ public Task BindModelAsync(ModelBindingContext bindingContext)
             WeakETag model = null;
             if (!string.IsNullOrWhiteSpace(suppliedWeakETag))
             {
-                model = WeakETag.FromWeakETag(suppliedWeakETag);
+                try
```

Comment:
Should this class have unit tests?

Reply:
Hmm I am not opposed to adding them in, but here is my thought process on why I left them out: we do have unit tests for ETag conversions in `WeakETagTests.cs`, and the integration test I'll add once PR [#2457](https://github.com/microsoft/fhir-server/pull/2457) is merged should ensure that a 400 is returned when an invalid version is passed in by the user.

I also noticed our other binder class (`PartialDateTimeBinder.cs`) doesn't have unit tests. Both binder classes only have the `BindModelAsync()` method, and I'm not confident that adding unit tests for it would add much value (I could be wrong, though).

Let me know your thoughts!

Reply:
Sounds good, I just wanted to make sure there was some test coverage for this.